src/components/LegalBanner.tsx

```diff
@@ -72,17 +82,24 @@ const LegalText: React.FC = () => (
   </Text>
 )
 
-const LegalBanner: React.FC<LegalBannerProps> = ({ startOpen, useFull, title }) => {
-  const [open, setOpen] = useState(startOpen)
+const DISCLAIMER = 'GP_DISCLAIMER'
+
+const LegalBanner: React.FC<LegalBannerProps> = ({ useFull, title }) => {
+  const [open, setOpen] = useState(localStorage.getItem(DISCLAIMER) || 'OPEN')
+
+  useEffect(() => {
+    open === 'OPEN' ? localStorage.setItem(DISCLAIMER, 'OPEN') : localStorage.setItem(DISCLAIMER, 'CLOSED')
```

Comment:
Being two the values, why not a boolean as a string?

```js
localStorage.setItem(IS_CLOSED_PARAM, 'true')
...
const [isClosed, setIsClosed] = useState(localStorage.getItem(IS_CLOSED_PARAM) === 'true')
...

```

Reply:
good rec, will work it in

Reply:
actually there are issues with the null type and the way it's stringified so for simplicities sake and the fact its working now I'll leave it as is :D if it's super annoying we can change it later

Reply:
But `localStorage.getItem(IS_CLOSED_PARAM) === 'true'` 

As I understand, `localStorage.getItem(IS_CLOSED_PARAM)` can be:
* `null`: `null == 'true'`   ---> `false` (no closed)
* `'true'`: `'true' == 'true'`   ---> `true` (closed)